modin/pandas/series.py

```diff
@@ -1135,21 +1135,6 @@ def keys(self):  # noqa: RT01, D200
         """
         return self.index
 
```

Comment:
Why delete `kurt`?

Reply:
Here, the base implementation is simply reused. The interface of `kurt` function is the same for dataframe and series.